requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyyaml==5.4
+dxpy
```

Comment:
should we set the version of dxpy as well?

Reply:
version added